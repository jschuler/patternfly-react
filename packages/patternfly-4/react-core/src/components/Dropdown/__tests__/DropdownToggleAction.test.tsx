// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { DropdownToggleAction } from '../DropdownToggleAction';

describe('DropdownToggleAction test', () => {
  it('DropdownToggleAction should match snapshot', () => {
    const view = shallow(
      <DropdownToggleAction
        className={"''"}
				isDisabled={false}
				onClick={() => {}}
				children={undefined}
				id={undefined}
				aria-label={undefined}
      />);
    expect(view).toMatchSnapshot();
  });
});
