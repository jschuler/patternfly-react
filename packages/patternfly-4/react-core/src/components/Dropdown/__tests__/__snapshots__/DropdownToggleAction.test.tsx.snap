// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownToggleAction test DropdownToggleAction should match snapshot 1`] = `
<button
  className="pf-c-dropdown__toggle-button ''"
  onClick={[Function]}
/>
`;
